src/Vault.hs

```diff
@@ -169,7 +179,7 @@ parseConfig fp = do
         else return defaultConfig
   where
     defaultConfig = Options "vaultaire" "vaultaire" "localhost"
-                            False False False 1000 2048 "" Broker
+                            False False False 1000 2048 "" "" Broker
```

Comment:
Yeah, hard to avoid. Maybe it'd be better split in to one option per line, allowing for inline comments?